talentsearch/resources/js/talentSearchConstants.ts

```diff
@@ -1,2 +1 @@
-export const BASE_URL = process.env.TALENTSEARCH_APP_URL as string;
 export const BASE_URL_DIR = process.env.TALENTSEARCH_APP_DIR as string;
```

Comment:
I know you probably wanted to minimize the amount of code change needed to achieve the solution but I wonder if this constant should be renamed from `BASE_URL_DIR` to `TALENTSEARCH_APP_DIR` to be consistent with the corresponding constant in admin and the .env file variable name.  This would also make it clear that the constant is a directory and not a URL.

Reply:
Yep, I like this suggestion.

Reply:
this is done. pls review

Reply:
I like it! 👍